DetectorDescription/RegressionTest/test/const_dump.cpp

```diff
@@ -81,10 +85,16 @@ int main(int argc, char* argv[]) {
       }
     }
 
-    std::vector<string> vnames;
-    DDVectorGetter::beginWith("Subdetector", vnames);
-    for (std::vector<string>::const_iterator sit = vnames.begin(); sit != vnames.end(); ++sit) {
-      std::cout << sit->c_str() << std::endl;
+    //This forces the 'vector' to be filled
+    cpv.lockdown();
```

Comment:
@ianna I did the `lockdown` here rather than right after the parsing in the case that we decide to clear the `DDVector` container in that call since such a clear would break the loop just before this one.